source/uwp/Renderer/lib/XamlBuilder.cpp

```diff
@@ -1710,6 +1731,92 @@ namespace AdaptiveNamespace
         return S_OK;
     }
 
+    template<>
+    void XamlBuilder::SetAutoSize<IEllipse>(IEllipse* destination, IInspectable* parentElement, IInspectable* imageContainer, bool mustHideElement)
```

Comment:
>mustHideElement [](start = 131, length = 15)

Can we name this variable something more descriptive? It took me a bit to figure out what this represents. Maybe reverse it and call it IsImageLoaded.